pkg/services/ngalert/notifier/channels/victorops_test.go

```diff
@@ -18,6 +18,9 @@ import (
 func TestVictoropsNotifier(t *testing.T) {
 	tmpl := templateForTests(t)
 
+	images, deleteFunc := newFakeImageStore(t)
```

Comment:
Do you need an actual on-disk fake for these tests, since they only test that a URL is present?

Reply:
Something like this?

```
func newFakeImageStore(t *testing.T, withFile bool) {
}
```